### PR TITLE
Support excluding packages during installation

### PR DESCRIPTION
**Pull request by psss**
This is related to #479.

### CODE REVIEW

File: spec/plans/prepare.fmf [hunk 1]
```diff
@@ -53,6 +53,9 @@ description: |
         Additionaly, the ``directory`` attribute can be used to
         install all packages from the given directory. Copr
         repositories can be enabled using the ``copr`` attribute.
+        Use the ``exclude`` option to skip selected packages from
+        installation.
```
Comment (thrix) [inline spec/plans/prepare.fmf:57]: Maybe we should mention the options are directly passed to the package manager, thus support globbing ...

File: tests/prepare/install/data/exclude.fmf [hunk 2]
```diff
@@ -0,0 +1,10 @@
+summary: Exclude packages during installation
+provision:
+    how: container
+prepare:
+    how: install
+    package: httpd-*
+    exclude: [httpd-manual]
```
[the remark this answers is not in the document]
Comment (psss) [reply tests/prepare/install/data/exclude.fmf:7]: It does support string as well. Shall I explicitly mention that?

**Review by hegerj — APPROVED** [review]

File: tests/prepare/install/data/exclude.fmf (hunk 2 above)
[the remark this answers is not in the document]
Comment (thrix) [reply tests/prepare/install/data/exclude.fmf:7]: I would just use it here, using `[something]` in yaml is a bit weird ... and even in this case, not needed right?

File: spec/plans/prepare.fmf (hunk 1 above)
Comment (psss) [reply spec/plans/prepare.fmf:57]: Added an explicit mention about globbing.

File: tests/prepare/install/data/exclude.fmf (hunk 2 above)
[the remark this answers is not in the document]
Comment (psss) [reply tests/prepare/install/data/exclude.fmf:7]: Test updated.

**Review by thrix — APPROVED** [review]
LGTM

**Comment by psss** [comment]
Thanks for the review!

**Pull request merged**